controllers/warehousesController.js

```diff
@@ -0,0 +1,108 @@
+const Joi = require("joi");
```

Comment:
is your new PR have two  features  SPF2-16  AND  SPF2-19

Reply:
yes

Reply:
great